Tools/clinic/clinic.py

```diff
@@ -5751,5 +5757,19 @@ def main(argv: list[str]) -> None:
 
 
 if __name__ == "__main__":
-    main(sys.argv[1:])
-    sys.exit(0)
+    cli = create_cli()
+    try:
+        args = cli.parse_args()
+        main(args)
```

Comment:
Just for symmetry with the other branches, more than anything else:

```suggestion
        main(args)
        sys.exit(0)
```